src/templates/admin/submission/manager/configurator.html

```diff
@@ -16,6 +18,7 @@
     <div class="box">
         <div class="row expanded">
             <div class="large-6 columns">
+                {% if user_is_journal_manager %}
```

Comment:
While this invocation will stop the inner HTML from rendering, it won't actually validate on the backend, we'd need validation to take place there.